code/datums/ai/ai_movement/hybrid_movement.dm

```diff
@@ -190,7 +190,7 @@
 				if(!(used_ref in falling_back))
 					falling_back |= used_ref
 					falling_back[used_ref] = TRUE
-				if(get_turf(movable_pawn) == next_step || (istype(next_step, /turf/open/transparent) && get_turf(movable_pawn) == GET_TURF_BELOW(next_step)))
+				if(get_turf(movable_pawn) == next_step || (istype(next_step, /turf/open) && get_turf(movable_pawn) == GET_TURF_BELOW(next_step)))
```

Comment:
I imagine this was meant to be for open space and will now break pathfinding due to not liking /turf/open